test/browser/features/support/env.rb

```diff
@@ -28,41 +28,17 @@ def get_error_message id
   skip_this_scenario unless Maze.driver.local_storage?
 end
 
-Before('@skip_ie_8') do |scenario|
-  browser = Maze.config.browser
-  skip_this_scenario unless browser != "ie_8"
-end
-
-Before('@skip_ie_9') do |scenario|
-  browser = Maze.config.browser
-  skip_this_scenario unless browser != "ie_9"
-end
+ERRORS = YAML::load_file('features/fixtures/browser_errors.yml')
 
-Before('@skip_ie_10') do |scenario|
-  browser = Maze.config.browser
-  skip_this_scenario unless browser != "ie_10"
-end
-
-Before('@skip_ie_11') do |scenario|
-  browser = Maze.config.browser
-  skip_this_scenario unless browser != "ie_11"
-end
-
-Before('@skip_safari_6') do |scenario|
-  browser = Maze.config.browser
-  skip_this_scenario unless browser != "safari_6"
-end
-
-Before('@skip_firefox_30') do |scenario|
-  browser = Maze.config.browser
-  skip_this_scenario unless browser != "firefox_30"
+ERRORS.keys.each do |browser|
+  Before("@skip_#{browser}") do |scenario|
+    skip_this_scenario if Maze.config.browser == browser
+  end
```

Comment:
Tidy

Reply:
Can thank @imjoehaines for the syntax on that one!